core-web/libs/block-editor/src/lib/nodes/video/video.node.ts

```diff
@@ -0,0 +1,110 @@
+import { mergeAttributes, Node } from '@tiptap/core';
+
+import { DotCMSContentlet } from '@dotcms/dotcms-models';
+
+declare module '@tiptap/core' {
+    interface Commands<ReturnType> {
+        videoBlock: {
+            setVideo: (attrs: DotCMSContentlet | string) => ReturnType;
+        };
+    }
+}
+
+export const VideoNode = Node.create({
+    name: 'video',
+
+    addAttributes() {
+        return {
+            src: {
+                default: null,
+                parseHTML: (element) => element.getAttribute('src'),
+                renderHTML: (attributes) => ({ src: attributes.src })
+            },
+            mineType: {
+                default: null,
+                parseHTML: (element) => element.getAttribute('mineType'),
+                renderHTML: (attributes) => ({ mineType: attributes.mineType })
+            },
+            width: {
+                default: null,
+                parseHTML: (element) => element.getAttribute('width'),
+                renderHTML: (attributes) => ({ width: attributes.width })
+            },
+            height: {
+                default: null,
+                parseHTML: (element) => element.getAttribute('height'),
+                renderHTML: (attributes) => ({ height: attributes.height })
+            },
+            data: {
+                default: null,
+                parseHTML: (element) => element.getAttribute('data'),
+                renderHTML: (attributes) => ({ data: attributes.data })
+            }
```

Comment:
Is this the same or almost the same structure as we do on images?

Reply:
They are different:

- `image` has the following attributes: `src`, `alt`, `title`, `href` (For images with link), and `data` (Contentlet information).

- `video` has the following attributes: `src`, `mineType `, `width `, `height `, and `data` (Contentlet information).

Reply:
Do we want a specific list of `attributes`?